doc/lua_api.txt

```diff
@@ -4995,6 +4995,19 @@ This is basically a reference to a C++ `ServerActiveObject`
           (default `{x=0, z=-2}`).
 * `get_clouds()`: returns a table with the current cloud parameters as in
   `set_clouds`.
+* `set_particle_overlay(parameters)`: configure a particle overlay on the client
+	* `parameters` is a table with the following fields:
+		* `name`: overlay name. It must be unique across all mods to prevent conflicts.
+		* `texture_name`: Texture used by particles
+		* `enabled`: Enable or disable the overlay
+		* `minpps`: Minimum particles spawned per second (max `20000`)
+		* `maxpps`: Maximum particles spawned per second (max `20000`)
+		* `direction`: Wind direction (between `0` and `359` degrees)
+		* `velocity`: Particle velocity applied on the defined direction (between `0.0`
+		  and `500.0`)
+		* `gravity_factor`: Physics gravity factor (between `0.0` and `100.0`)
+		* `size_x`: Texture width scale factor (default `1.0`)
+		* `size_y`: Texture height scale factor (default `1.0`)
```

Comment:
Ditto. Tabs are superior, but here we use spaces.
Also: The return value needs documentation